frontend/src/views/Login.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 <div class="auth-page">
-    <AppAuthForm v-if="!loading"
+    <AppAuthForm v-if="!loading && !isLoggedIn"
```

Comment:
? I thought we were getting rid of this.

Reply:
yes this is redundant. removed

Reply:
cool